Source/player.cpp

```diff
@@ -224,22 +225,22 @@ int PlayerStruct::GetMaximumAttributeValue(CharacterAttribute attribute) const
 	return MaxStats[static_cast<std::size_t>(_pClass)][static_cast<std::size_t>(attribute)];
 }
 
-void PlayerStruct::UpdateTargetPosition()
+SDL_Point PlayerStruct::GetTargetPosition() const
 {
 	// clang-format off
-	int directionOffsetX[8] = {  0,-1, 1, 0,-1, 1, 1,-1 };
-	int directionOffsetY[8] = { -1, 0, 0, 1,-1,-1, 1, 1 };
+	constexpr int directionOffsetX[8] = {  0,-1, 1, 0,-1, 1, 1,-1 };
+	constexpr int directionOffsetY[8] = { -1, 0, 0, 1,-1,-1, 1, 1 };
 	// clang-format on
-	_ptargx = _pfutx;
-	_ptargy = _pfuty;
+	SDL_Point target = { _pfutx, _pfuty };

```

Comment:
```suggestion
	SDL_Point target { _pfutx, _pfuty };

```

Reply:
If you don't mind me asking, what is the name of this "pattern/feature"? I've seen it mentioned in a few places saying something like "no assignments".